### ISSUE TITLE
Launchpad repo?

### ISSUE THREAD

**Issue by SamuelMarks** [deric/mesos-deb-packaging#16]
I note here that you haven't succeeded in getting it on apache.mesos.org: https://issues.apache.org/jira/browse/MESOS-74

So I was thinking of starting up a launchpad repository, although this sounds like something that would be better off maintained by the original author, i.e.: you.

Would you be interested in setting up one?

Thanks for your consideration

**Comment by SamuelMarks** [comment]
Ah nevermind, found this blocker => https://github.com/jordansissel/fpm/issues/170

**Comment by deric** [comment]
Hey, thanks for interest in this repo!  It contains a bit hacky scripts, which wasn't meant for a general usage and became way too popular :) Anyway, it would be much preferable to make significant changes to this packaging:
* separate packages for master, slave, java and python bindings
* make it as proper Debian package, an archive that can be uploaded e.g. to launchpad

I've already started working on this, however proper testing requires a physical machine (not just virtual LXC containers) with clean Debian distribution. I'll upload this new versions for anyone who would like to participate. We've already discussed this on mesos users mailing list, hopefully someone else might contribute as well.
